src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -78,6 +78,8 @@
             <argument type="service" id="translator" />
             <argument type="service" id="translation.reader" />
             <argument type="service" id="translation.extractor" />
+            <argument>%translator.default_path%</argument>
+            <argument on-invalid="null">%twig.default_path%</argument>
```

Comment:
`on-invalid="null"` over container parameter is not supported or (surely) I'm missing something? Should I do it into `FrameworkExtension` then?

Reply:
yes, it should be done in the DI extension then